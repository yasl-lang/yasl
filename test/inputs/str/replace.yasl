echo 'YASL'->replace('T', 'S')
echo 'YASL'->replace('SL', '')
echo 'AAA'->replace('A', 'B', 0)
echo 'AAA'->replace('A', 'B', 1)
echo 'AAA'->replace('A', 'B', 2)
echo 'AAA'->replace('A', 'B', 3)
echo 'AAA'->replace('A', 'B', 4)
echo ('abbc'->replace('b', 'B'))
