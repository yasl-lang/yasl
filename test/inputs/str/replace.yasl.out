YASL, 0
YA, 1
AAA, 0
BAA, 1
CBCBBABBA, 2
CBCBCBA, 3
BBB, 3
aBBc
